internal/locale/locale.go

```diff
@@ -45,6 +45,7 @@ func init() {
 	if locale == "" {
 		cfg, err := config.New()
 		if err == nil {
+			defer cfg.Close()
```

Comment:
Should we use the closer here so any close errors are logged?

Reply:
This is an init function, so logging may not be available. I'd rather not risk making this a bigger issue than it needs to be, especially considering long term we'd be getting rid of this init function anyway.